Makefile

```diff
@@ -0,0 +1,19 @@
+# Must have `sentry-cli` installed globally
+# Following variable must be passed in
+# SENTRY_AUTH_TOKEN
+SENTRY_ORG=sentry-test
+SENTRY_PROJECT=fullstory-demo
```

Comment:
Maybe all of these values can be put in a `.env` file so they can be used by the CLI and by the Create React App as well?

Reply:
@patrick-fs That's what I would prefer but the Makefile doesn't automatically load values from the `.env` (but the Sentry CLI will pick it up). You have to write code that opens the file and loads it. I prefer keeping this simple even if I have to repeat a few lines of config.